### ISSUE TITLE
New Feature Face-ID Adapter for Control Face animation in the Multiple Faces image is availabled 🔥🔥🔥🔥🔥

### ISSUE THREAD

**Issue by aihacker111** [aihacker111/Efficient-Live-Portrait#13]
# Check List:
+ This new update is using Face-ID AcrFace to control Face Animations in the Multiples Images 
+ I'm also update new TensorRT run on latest TensorRT version
+ Project can run on Google Colab for demo, I'll update Gradio Interface later 

+ In many near day ago, I'm assigned to the new research paper, so I cannot update new feature faster, but now the model is came to the training process, so I come back to update new idea on this great project 
Thank you and see result using Face-ID Adapter in Live-Portrait

+ Checkout here for run demo: https://github.com/aihacker111/Efficient-Live-Portrait

# REFINE IMAGE: 
![368220873_826368889022136_4472311944594836999_n](https://github.com/user-attachments/assets/e1dfd5d1-12d3-4a3a-bf7e-cfd2125fe3eb)
# Result Video control Face-Animation

https://github.com/user-attachments/assets/bc47d894-a984-4bce-8851-4921b40c7480

**Comment by x4080** [comment]
Hi, I cannot seems to play the video ? Is it only me ?

BTW did you implement the video 2 video yet ?

**Comment by aihacker111** [comment]
@x4080 seem like only you cannot open video , v2v , I’m plan for this

**Comment by aihacker111** [comment]
@x4080 BTW , I’m also work on multiple object in driving video , because the official is allow only 1 face video , but we can do more than 1 face

**Comment by municef1** [comment]
Thanks to your great work!

+ How much difference is there in inference time compared to the existing model pipeline in practical terms?

**Comment by aihacker111** [comment]
@municef1 TensorRT FP16 is work with 374 frame is 30 second and fp32 is 87s , result of fp32 is better than fp16

**Comment by x4080** [comment]
> @x4080 BTW , I’m also work on multiple object in driving video , because the official is allow only 1 face video , but we can do more than 1 face

Cool ! Cant wait for it

**Comment by aihacker111** [comment]
@x4080 support me in the adding audio bro, I'm take time to fixing the output, seem like it cannot return out of the loop

**Comment by aihacker111** [comment]
@x4080 Check out this my friend 

https://github.com/user-attachments/assets/23fe2d47-0b26-473e-8e87-a908bbe1fd19


https://github.com/user-attachments/assets/e4372745-3d30-456e-8879-9d16f110d532

**Comment by aihacker111** [comment]
@x4080  Video2Video is now availabled, you can use it, very funny

**Comment by x4080** [comment]
@aihacker111 Thanks, very cool man

**Comment by aihacker111** [comment]
@x4080 Hey man, I'm update controlnet open-pose feat sdxl lightning text+image to image feature, you can try on this bro

**Comment by x4080** [comment]
@aihacker111 yes thats very good, is it photo to sdxl using openpose then live portrait ?
